xbmc/music/windows/GUIWindowMusicPlaylistEditor.cpp

```diff
@@ -155,6 +155,12 @@ bool CGUIWindowMusicPlaylistEditor::GetDirectory(const std::string &strDirectory
     db->m_bIsShareOrDrive = true;
     items.SetPath("");
     items.Add(db);
+    CFileItemPtr vdb(new CFileItem("videodb://musicvideos/", true));
```

Comment:
Please add an empty line (and maybe also one in the existing code between "files://" and "musicdb://") for better readability.